AmericanMedicalAssociation/AMA-style-guide
# AMA-122: Standard News item and list responsive implementation

https://palantir.atlassian.net/browse/AMA-122

Adds responsive implementation for the News Item Standard Row and News Standard list based on the behaviors on the Corp site.

### Steps to test

1. Navigate to Molecules > Components >News Item Standard Row
2. Observe that the News Item Standard Row is present and responsive.
3. Navigate to Molecules > Components > News Label
4. Observe that the News Label is present and responsive.
5. Navigate to Organisms > Sections > News Standard
6. Observe that the News Standard list is present and responsive.
7. Navigate to Templates > Home > Home
8. Observe that the News Standard list is included in the home template and is responsive.
